Add `Unlock_requirements/Hush-hush` wiki article (#10702)
ppy/osu-wiki: wiki/Medals/Unlock_requirements/Hush-hush/en.md

# Unlock requirements for Hush-Hush medals

> *Note: The actual solutions of [Hush-Hush](/wiki/Medals#hush-hush) medals are intentionally omitted from this wiki to help preserve their secrecy.*

Reviewer: ```suggestion
The actual solutions of [Hush-Hush](/wiki/Medals#hush-hush) medals are intentionally omitted from this wiki to help preserve their secrecy. The community is encouraged to work together in order to find them.

There are, however, some general rules that the solutions tend to follow:
```

> *Note: While some conditions for Hush-Hush medals are known, their solutions are intentionally omitted from this wiki to help preserve their secrecy.*

Reviewer 2: ```suggestion
*Note: The actual solutions of [Hush-Hush](/wiki/Medals#hush-hush) medals are intentionally omitted from this wiki to help preserve their secrecy.*
```

> - A medal only appears in the profile of [gamemodes](/wiki/Game_mode) it can be obtained in.

Reviewer: ```suggestion
- A medal only appears in the profile of [game modes](/wiki/Game_mode) it can be obtained in.
```

> - A medal only appears in the profile of [gamemodes](/wiki/Game_mode) it can be obtained in. However, there are exceptions due to various reasons, such as star rating changes.

Reviewer: what do star rating changes have to do with which game mode a medal is available on?

Author: > what do star rating changes have to do with which game mode a medal is available on?

There are multiple medals that have star rating requirement (for example need to be played on 4*+), but in specific gamemodes such high star rating may not exist even with mods applied, making it impossible to get in said gamemode.

Reviewer 2: this doesn't sound like a real thing, likely some kind of coincidence?
```suggestion
- A medal only appears in the profile of [gamemodes](/wiki/Game_mode) it can be obtained in.
```

Author: > this doesn't sound like a real thing, likely some kind of coincidence?

I mean, yea, I believe it's not intentional, but that happens relatively often, when all 4 gamemodes are not considered properly. Also reminds me of somewhat recent incident with Not Bluffing not working (Ephemeral fixed it later) after osu! star rating changes.

Reviewer 2: i don't think this should account for unexpected/unintentional bugged cases, best to keep it straightforward.

> - If a medal does not appear in a gamemode's profile, this medal cannot be obtained in that particular gamemode. If said medal is shown in all gamemodes' profiles, in most cases it means you can obtain it in any gamemode, but there are exceptions due to various reasons, such as star rating changes.

Reviewer 3: ```suggestion
- A medal only appears in a [game mode](/wiki/Game_mode)'s profile it can be obtained in. However, there are exceptions due to various reasons, such as star rating changes.
```

Author: ```suggestion
- A medal only appears in the profile of [gamemodes](/wiki/Game_mode) it can be obtained in. However, there are exceptions due to various reasons, such as star rating changes.

```

- Medals can only be obtained on beatmaps that are either [Ranked](/wiki/Beatmap/Category#ranked), [Loved](/wiki/Beatmap/Category#loved), [Approved](/wiki/Beatmap/Category#approved) or [Qualified](/wiki/Beatmap/Category#qualified).

> - Beatmap-specific medals are always related to songs from [Featured Artists](/wiki/People/Featured_Artists). This is a convention which has been emphasized more recently, so some older medals may be an exception to this rule.

Reviewer: ```suggestion
- Beatmap-specific medals are always related to songs from [Featured Artists](/wiki/People/Featured_Artists). This is a convention which has been emphasised more recently, so some older medals may be an exception to this rule.
```

> - Map-specific medals, which can only be obtained on certain beatmaps, are always related to songs from [Featured Artists](/wiki/People/Featured_Artists). This is a convention which has been emphasized more recently, so some older medals are an exception to this rule.

Reviewer 2: ```suggestion
- Beatmap-specific medals are always related to songs from [Featured Artists](/wiki/People/Featured_Artists). This is a convention which has been emphasized more recently, so some older medals may be an exception to this rule.
```

> - For map-specific medals, which can only be obtained on certain beatmap(s) - required beatmap(s) artist will always be a [Featured Artist](/wiki/People/Featured_Artists/).

Reviewer 3: ```suggestion
- Map-specific medals, meaning they can only be obtained on certain beatmaps, are always relating to songs from [Featured Artists](/wiki/People/Featured_Artists). This is a convention which was introduced recently, so earlier medals are an exception to this rule.
```

Author: ```suggestion
- Map-specific medals, which can only be obtained on certain beatmaps, are always related to songs from [Featured Artists](/wiki/People/Featured_Artists). This is a convention which has been emphasized more recently, so some older medals are an exception to this rule.
```

> - Medals in the "Hush-Hush (Expert)" subcategory do not allow usage of any [difficulty reduction](/wiki/Gameplay/Game_modifier#difficulty-reduction) or [special](/wiki/Gameplay/Game_modifier#special) mods.

Reviewer 2: expert medals only impose an explicit limitation on diff reduction mods
```suggestion
- Medals in the "Hush-Hush (Expert)" subcategory do not allow usage of any [difficulty reduction](/wiki/Gameplay/Game_modifier#difficulty-reduction) mods.
```

> - Medals in subcategory called "Hush-Hush (Expert)" do not allow usage of any [difficulty reduction](/wiki/Gameplay/Game_modifier#difficulty-reduction) or [special](/wiki/Gameplay/Game_modifier#special) mods.

Reviewer 3: ```suggestion
- Medals in the subcategory called "Hush-Hush (Expert)" do not allow usage of any [difficulty reduction](/wiki/Gameplay/Game_modifier#difficulty-reduction) or [special](/wiki/Gameplay/Game_modifier#special) mods.
```

# Unlock requirements for Hush-Hush medals

*Note: While some conditions for Hush-Hush medals are known, their solutions are intentionally omitted from this wiki to help preserve their secrecy.*

> - A medal only appears in the profile of [gamemodes](/wiki/Game_mode) it can be obtained in. However, there are exceptions due to various reasons, such as star rating changes.

Reviewer 3: ```suggestion
- On the osu! website, a medal appears in the user profile only for [game modes](/wiki/Game_mode) it can be obtained in. However, there are exceptions due to various reasons, such as star rating changes.
```

`game mode` are two words as far as I know.

Author: afaik there is no other gamemode profile apart from osu-web one, so what's even the point to state it

Reviewer: this would also apply to the in-game profile (lazer) so yes it's a bit pointless to state

- Medals can only be obtained on beatmaps that are either [Ranked](/wiki/Beatmap/Category#ranked), [Loved](/wiki/Beatmap/Category#loved), [Approved](/wiki/Beatmap/Category#approved) or [Qualified](/wiki/Beatmap/Category#qualified).
- Map-specific medals, which can only be obtained on certain beatmaps, are always related to songs from [Featured Artists](/wiki/People/Featured_Artists). This is a convention which has been emphasized more recently, so some older medals are an exception to this rule.

> - Medals in the subcategory called "Hush-Hush (Expert)" do not allow usage of any [difficulty reduction](/wiki/Gameplay/Game_modifier#difficulty-reduction) or [special](/wiki/Gameplay/Game_modifier#special) mods.

Reviewer 3: ```suggestion
- Medals in the "Hush-Hush (Expert)" subcategory do not allow usage of any [difficulty reduction](/wiki/Gameplay/Game_modifier#difficulty-reduction) or [special](/wiki/Gameplay/Game_modifier#special) mods.
```

Maybe an option 🤔

Author: not much of a difference imo
